azure-mgmt-servicebus/src/main/java/com/microsoft/azure/management/servicebus/CheckNameAvailabilityResult.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.servicebus;
+
+import com.microsoft.azure.management.servicebus.implementation.CheckNameAvailabilityResultInner;
+
+/**
+ * The {@link com.microsoft.azure.management.servicebus.Namespaces#checkNameAvailability} action result.
+ */
+public class CheckNameAvailabilityResult {
+    private CheckNameAvailabilityResultInner inner;
+    /**
+     * Creates an instance of the check name availability result object.
+     *
+     * @param inner the inner object
+     */
+    public CheckNameAvailabilityResult(CheckNameAvailabilityResultInner inner) {
+        this.inner = inner;
+    }
+    /**
+     * @return a boolean value that indicates whether the name is available for
+     * you to use. If true, the name is available. If false, the name has
+     * already been taken or invalid and cannot be used.
+     */
+    public boolean isAvailable() {
+        return inner.nameAvailable();
+    }
+    /**
+     * @return the reason that a namespace name could not be used. The
+     * Reason element is only returned if NameAvailable is false. Possible
+     * values include: 'AccountNameInvalid', 'AlreadyExists'.
+     */
+    public UnavailableReason reason() {
```

Comment:
This is a generated type from the swagger. If we rename then it gets overwritten in the next refresh. What do you think?